components/NavigationBar/index.js

```diff
@@ -5,33 +5,35 @@ import { FaHouse, FaPlus, FaChartPie } from "react-icons/fa6";
 function NavigationBar() {
   const router = useRouter();
 
-  const isActive = (path) => {
-    return router.pathname === path;
-  };
+ function isActive(path) {
+   return router.pathname === path;
+ }
 
-  return (
-    <Styled.NavBar>
-      <Styled.NavContainer>
-        <Styled.NavLink onClick={() => router.push("/")} active={isActive("/")}>
-          <FaHouse />
-        </Styled.NavLink>
+ return (
+   <Styled.NavBar>
+     <Styled.NavContainer>
+       <Styled.NavLink key="home" href="/" active={isActive("/")}>
+         <FaHouse />
+       </Styled.NavLink>
 
-        <Styled.NavLink
-          onClick={() => router.push("/newentry")}
-          active={isActive("/newentry")}
-        >
-          <FaPlus />
-        </Styled.NavLink>
+       <Styled.NavLink
+         key="newentry"
+         href="/newentry"
+         active={isActive("/newentry")}
```

Comment:
this still won't work, since you need to change the prop name here to $active. Also, a key is not needed here, since you are not using 'map'.